k8s-readonly.tf

```diff
@@ -205,12 +205,12 @@ resource "kubernetes_service" "hms_readonly" {
       port        = 9083
       target_port = 9083
     }
-    type                        = "LoadBalancer"
-    load_balancer_source_ranges = var.ingress_cidr
+    type                        = var.enable_vpc_endpoint_services ? "LoadBalancer" : "ClusterIP"
+    load_balancer_source_ranges = var.enable_vpc_endpoint_services ? var.ingress_cidr : null
```

Comment:
Does the else of this ternary need to be `[]` instead of null or will setting null here result in k8s ignoring this entirely?

Reply:
null essentiall will mean we are setting `load_balancer_source_ranges` , else I think k8s complains saying that `load_balancer_source_ranges ` invalid for ClusterIP type.